easybuild/easyconfigs/a/ALPS/ALPS-2.3.0-CrayGNU-19.06-Boost-1.69.0-python2.eb

```
@@ -3,28 +3,29 @@ easyblock = 'CMakeMake'
 
 name = 'ALPS'
 version = "2.3.0"
-versionsuffix = "-Boost-1.65.1-python2"
+versionsuffix = "-Boost-1.69.0-python2"
 
 homepage = 'http://alps.comp-phys.org'
 description = """The ALPS project (Algorithms and Libraries for Physics
 Simulations) is an open source effort aiming at providing high-end simulation
 codes for strongly correlated quantum mechanical systems as well as C++
 libraries for simplifying the development of such code."""
 
-toolchain = {'name': 'CrayGNU', 'version': '18.08'}
+toolchain = {'name': 'CrayGNU', 'version': '19.06'}
 toolchainopts = {'pic': True, 'dynamic': True}
```

Dynamic build is the default behaviour of EB, so this tc opt can be omitted.  
```suggestion
toolchainopts = {'pic': True}
```